doc/index.rst

```diff
@@ -150,6 +151,9 @@ Optional environment variables
 
 Note that some of these currently tied only to a single option, but we still leave them for the future when more options become available (e.g. ament with BUILDER).
 
+* `ABICHECK_MERGE` (default: not set): Used only when `ABICHECK_URL` is set. For travis it can be set to 'auto' to auto-detect pull requests. If set to 'true' the merge parent (see `Terminology section <#terminology>`_) will be checked against.
+* `ABICHECK_URL` (default: not set): Run binary compatibility check with `ABICC <https://github.com/lvc/abi-compliance-checker>`_. The URL should point to a baseline archive (\*.tar.\*,\*.zip, \*.tgz or \*.tbz2), see more in `the ABI checks section <#abi-checks>`_)
```

Comment:
`ABICHECK_URL` points to a branch in [this example](https://github.com/ros-industrial/industrial_ci/pull/199/files#diff-0ced3dfd44ba156272e6db34a5247462R282), not a tag. If I'm not misunderstood, please add explanation here (or somewhere appropriate).

Reply:
IMHO setting it to a branch is the best option for maintenance, but this requires a stable branch.
The maintainer can choose any baseline: Some might choose the latest release, some might choose the first release for a certain distro.

Reply:
For clarification:  IMHO setting it to a *branch-based URL* is the best option for maintenance.